src/consistency.test.ts

```diff
@@ -0,0 +1,270 @@
+        response: new Response(null, { headers }),
+        url: `${uncachedEdgeURL}/${siteID}/production/${key}`,
+      })
+      .get({
+        headers: { authorization: `Bearer ${edgeToken}` },
+        response: new Response(value, { headers }),
+        url: `${uncachedEdgeURL}/${siteID}/production/${key}`,
+      })
+
+    globalThis.fetch = mockStore.fetch
+
+    const context = {
+      edgeURL,
+      siteID,
+      token: edgeToken,
+      uncachedEdgeURL,
+    }
+
+    env.NETLIFY_BLOBS_CONTEXT = Buffer.from(JSON.stringify(context)).toString('base64')
+
+    const blobs = getStore('production')
+
+    const data = await blobs.get(key, { consistency: 'strong' })
+    expect(data).toBe(value)
+
+    const meta = await blobs.getMetadata(key, { consistency: 'strong' })
+    expect(meta?.etag).toBe(headers.etag)
+    expect(meta?.metadata).toEqual(mockMetadata)
+
+    const dataWithMeta = await blobs.getWithMetadata(key, { consistency: 'strong' })
+    expect(dataWithMeta?.data).toBe(value)
+    expect(dataWithMeta?.etag).toBe(headers.etag)
+    expect(dataWithMeta?.metadata).toEqual(mockMetadata)
+
+    expect(mockStore.fulfilled).toBeTruthy()
+  })
+
+  test('Respects the consistency mode supplied in the `getStore()` constructor', async () => {
+    const mockMetadata = {
+      name: 'Netlify',
+      cool: true,
+      functions: ['edge', 'serverless'],
+    }
+    const headers = {
+      etag: '123456789',
+      'x-amz-meta-user': `b64;${base64Encode(mockMetadata)}`,
+    }
+    const mockStore = new MockFetch()
+      .get({
+        headers: { authorization: `Bearer ${edgeToken}` },
+        response: new Response(value),
+        url: `${uncachedEdgeURL}/${siteID}/production/${key}`,
+      })
+      .head({
+        headers: { authorization: `Bearer ${edgeToken}` },
+        response: new Response(null, { headers }),
+        url: `${uncachedEdgeURL}/${siteID}/production/${key}`,
+      })
+      .get({
+        headers: { authorization: `Bearer ${edgeToken}` },
+        response: new Response(value, { headers }),
+        url: `${uncachedEdgeURL}/${siteID}/production/${key}`,
+      })
+
+    globalThis.fetch = mockStore.fetch
+
+    const blobs = getStore({
+      consistency: 'strong',
+      edgeURL,
+      name: 'production',
+      token: edgeToken,
+      siteID,
+      uncachedEdgeURL,
+    })
+
+    const data = await blobs.get(key)
+    expect(data).toBe(value)
+
+    const meta = await blobs.getMetadata(key)
+    expect(meta?.etag).toBe(headers.etag)
+    expect(meta?.metadata).toEqual(mockMetadata)
+
+    const dataWithMeta = await blobs.getWithMetadata(key)
+    expect(dataWithMeta?.data).toBe(value)
+    expect(dataWithMeta?.etag).toBe(headers.etag)
+    expect(dataWithMeta?.metadata).toEqual(mockMetadata)
+
+    expect(mockStore.fulfilled).toBeTruthy()
+  })
+
+  test.only('Respects the consistency mode supplied in the `getDeployStore()` constructor', async () => {
```

Comment:
```suggestion
  test('Respects the consistency mode supplied in the `getDeployStore()` constructor', async () => {
```

Reply:
Oops! Fixed in https://github.com/netlify/blobs/pull/145/commits/eccd51fbd35d12ebb85e67627f1c5fb5e191cdfe.